index.php

```diff
@@ -243,7 +246,7 @@
     <div class="col-sm-6 pt-1 pt-md-0">
       <div class="card text-center border-info">
         <div class="card-body">
-          <h5 class="card-title"><i class="bi bi-hdd"></i>&nbsp;SD Card</h5>
+          <h5 class="card-title"><i class="bi bi-hdd"></i>&nbsp;Disc Space</h5>
```

Comment:
Small typo, this should be "Dis**k** Space".

Reply:
Will be corrected.